docs/html/development/getting-started.rst

```diff
@@ -40,6 +40,13 @@ To run tests locally, run:
 
     $ tox -e py36
 
+Generally, it takes long time to run pip's test suite. To run tests in parallel,
```

Comment:
I would say "it can take a long time" instead of "it takes long time"

Reply:
done.